client/landing/gutenboarding/onboarding-block/beta-opt-in/index.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+	Title,
+	SubTitle,
+	ActionButtons,
+	NextButton,
+	BackButton,
+	SkipButton,
+} from '@automattic/onboarding';
+import { useDispatch } from '@wordpress/data';
+import { Icon, siteLogo, header, navigation, arrowRight } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import React from 'react';
+import useStepNavigation from '../../hooks/use-step-navigation';
+import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
+
+import './style.scss';
+
+const BetaOptIn: React.FunctionComponent = () => {
+	const { __ } = useI18n();
+	const { goNext, goBack } = useStepNavigation();
+	const { enrollInFseBeta } = useDispatch( ONBOARD_STORE );
+	const pickBeta = ( shouldEnroll: boolean ) => {
+		enrollInFseBeta( shouldEnroll );
+		goNext();
+	};
+	return (
+		<div className="gutenboarding-page beta-opt-in">
+			<div className="beta-opt-in__header">
+				<div>
+					<Title>{ __( 'Join Our Full Site Editing Beta' ) }</Title>
+					<SubTitle>{ __( 'Experience the new WordPress.com site editing features' ) }</SubTitle>
+				</div>
+				<ActionButtons className="beta-opt-in__actions">
+					<BackButton onClick={ goBack } />
+					<SkipButton onClick={ () => pickBeta( false ) }>{ __( 'No Thanks' ) }</SkipButton>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **38** times:
`translate( 'No, thanks.' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/fse-beta-in-signup)